helix-contract/contracts/mapping-token/v2/erc20-mapping-protocol/Erc20Sub2SubMigration.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.10;
+import "@zeppelin-solidity-4.4.0/contracts/utils/math/SafeMath.sol";
+import "../../interfaces/IERC20.sol";
+
+contract Erc20Sub2SubMigration {
```

Comment:
should we add an function for burn?

Reply:
Ref:

https://github.com/darwinia-network/darwinia-messages-sol/blob/master/contracts/tokens/contracts/ds-token/token.sol#L427